broker-cli/broker-daemon-client/config.js

```diff
@@ -15,6 +22,13 @@ try {
    */
   var USER_CONFIG = require(path.resolve(os.homedir(), '.sparkswap.js'))
 } catch (e) {
+  // We will only trigger an error if the `~/.sparkswap.js` configuration file
+  // exists on the user's machine. Otherwise we just silently set to a default
+  // configuration.
+  if (e.message && !e.message.includes(MISSING_FILE_MESSAGE)) {
```

Comment:
let's put this into a function so we can test it

Reply:
@treygriffith added tests